src/bijectors/rational_quadratic_spline.jl

```diff
@@ -100,8 +100,8 @@ function RationalQuadraticSpline(
     widths::A, heights::A, derivatives::A, B::T2
 ) where {T1,T2,A<:AbstractVector{T1}}
     return RationalQuadraticSpline(
-        (cumsum(vcat([zero(T1)], LogExpFunctions.softmax(widths))) .- 0.5) * 2 * B,
-        (cumsum(vcat([zero(T1)], LogExpFunctions.softmax(heights))) .- 0.5) * 2 * B,
+        cumsum(vcat([zero(T1)], LogExpFunctions.softmax(widths))) * 2 * B .- B,
```

Comment:
We could save a few allocations:
```suggestion
        cumsum(vcat([zero(T1)], LogExpFunctions.softmax(widths))) .* (2 * B) .- B,
```